db/migrate/20201127094738_create_placement_need_profiles.rb

```diff
@@ -0,0 +1,23 @@
+class CreatePlacementNeedProfiles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :placement_need_profiles do |t|
```

Comment:
Why is it called `profile`? Wouldn't `placement_needs` be enough and accurate?

Reply:
Good idea